### PR TITLE
Add a synagogue

### PR DESCRIPTION
**Pull request by Standing-Storm**
#### Summary
Content "Add a synagogue"

#### Purpose of change

There are churches but no synagogues, even though New England has a higher number of Jews than the American population average.  Time to change that. 

#### Describe the solution

Add a synagogue building.  It's the more traditional bimah in the center style, because that's the way my real-life synagogue is so I prefer it, plus it helps further differentiate it from the church. It also adds several relevant Judaica objects that don't currently exist: Torah, tallitot (gadol and katan), tefillin, etc, so your character can properly perform _shacharit_, as well as a number of Judaica books.

#### Testing

Synagogue spawns correctly and items are in the right place.  It's much rare than churches (just like real life)--some brief testing came up about once every other overmap on default city density. 

#### Additional context

 How can my character have the synagogue he attends and the synagogue he would rather be _eaten by zombies_ than set foot into if there are no synagogues?

The magiclysm tag is because I added a tallit gadol to the starting gear of the Magiclysm rabbi.

### CODE REVIEW

**Comment by amoteego** [comment]
This seems interesting! 
Will there be a ner tamid? Will it remain lit despite the loss of the power grid, perhaps from some sort of emergency generator or connection to solar power?

**Comment by Standing-Storm** [comment]
> This seems interesting! Will there be a ner tamid? Will it remain lit despite the loss of the power grid, perhaps from some sort of emergency generator or connection to solar power?

I was actually thinking about how to do that earlier today, but I don't have a good answer to that yet.  My brief bit of research indicated that a lot of synagogues don't really have a ner tamid contingency for even a normal power outage, much less the end of the world (mine certainly isn't prepared for either case). 

One idea I had was to have a chance of oil lamp spawn near the Aron Hakodesh, just as a sad example of all the tragedy that has befallen the post-Cataclysmic world.

**Comment by Standing-Storm** [comment]
Shacharit time
image

**Comment by Standing-Storm** [comment]
I'm not sure what the project's guidelines are for non-English word plurals.  I'm sincerely not sure what the English plural even would be for "tallit gadol" ("tallits gadol?) since everyone I talk uses the Hebrew terms when talking about them.

**Comment by Procyonae** [comment]
> I'm not sure what the project's guidelines are for non-English word plurals. I'm sincerely not sure what the English plural even would be for "tallit gadol" ("tallits gadol?) since everyone I talk to uses the Hebrew terms when talking about them.

If you can't find anything from a quick google you can just make them str_sp ig, someone can always come along and um actually it later and the singular being used incorrectly seems better than guessing plurals

**Comment by Procyonae** [comment]
Had a look in game and it looks nice ^^ The only thing I'd suggest is maybe adding a gutter drop so you can climb up and see the raised section of roof you added but it's cool if you don't want peeps climbing on there

**Comment by Standing-Storm** [comment]
> Had a look in game and it looks nice ^^ The only thing I'd suggest is maybe adding a gutter drop so you can climb up and see the raised section of roof you added but it's cool if you don't want peeps climbing on there

Oh, that is a good idea. That's just an honest oversight on my part.

File: data/json/mapgen/synagogue.json [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "palettes": [ "roof_palette" ],
+      "terrain": { ".": "t_shingle_flat_roof", "<": "t_stairs_up", "#": "t_wall_w", "o": "t_open_air_rooved" }
```
Comment (Procyonae) [inline data/json/mapgen/synagogue.json:156]:
```suggestion
      "terrain": { ".": "t_shingle_flat_roof" }
```

File: data/json/mapgen/synagogue.json [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        "   -................-   ",
+        "   --..............--   ",
+        "    -..............-    ",
+        "    -..............-    ",
+        "    --............--    ",
+        "     ------------d-     ",
+        "                        "
+      ],
+      "palettes": [ "roof_palette" ],
+      "terrain": { ".": "t_tile_flat_roof", "|": "t_brick_wall", "o": "t_open_air_rooved", "w": "t_window", "d": "t_gutter_drop" }
```
Comment (Procyonae) [inline data/json/mapgen/synagogue.json:120]:
```suggestion
        "     ------------5-     ",
        "                        "
      ],
      "palettes": [ "roof_palette" ],
      "terrain": { ".": "t_tile_flat_roof", "|": "t_brick_wall", "o": "t_open_air_rooved", "w": "t_window" }
```

**Review by Procyonae — COMMENTED** [review]
Sorry I'm the roof police c;

**Comment by Procyonae** [comment]
Is there any reason you didn't go with the same colour as church for the overmap symbol btw?

**Comment by Standing-Storm** [comment]
> Is there any reason you didn't go with the same colour as church for the overmap symbol btw?

Blue symbolizes holiness in Judaism--it's the color of the sky and the sea (and thus the majesty of G-d's creation), and blue dye was traditionally used to color the vestments of the high priest and the fringes attached to the corners of garments, in addition to the Middle-East-wide belief in blue repelling the evil eye (see the hamsa, nazar, etc)

I play with Larwick Overmap, though, , so if a blue S is already some common or important symbol there's always light blue or cyan.

**Comment by Procyonae** [comment]
> > Is there any reason you didn't go with the same colour as church for the overmap symbol btw?
> 
> Blue symbolizes holiness in Judaism--it's the color of the sky and the sea (and thus the majesty of G-d's creation), and blue dye was traditionally used to color the vestments of the high priest and the fringes attached to the corners of garments, in addition to the Middle-East-wide belief in blue repelling the evil eye (see the hamsa, nazar, etc)
> 
> I play with Larwick Overmap, though, so if a blue S is already some common or important symbol there's always light blue or cyan.

Nope blue isn't used for much just wanted to check there was reasoning ^-^

**Comment by Procyonae** [comment]
I think you want superceded -> superseded and encyclopediac -> encyclopedic (or encyclopædic bc æ best character (don't really)) and then add the rest to the dictionary ideally so it doesn't flag them next time someone else touches any of this

**Pull request merged**